### PR TITLE
Hashmap repeated word

### PR DESCRIPTION
**Pull request by katrina-l-hill**

### CODE REVIEW

**Comment by katrina-l-hill** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: python/code_challenges/hashtable_repeated_word.py [hunk 1]
```diff
@@ -2,21 +2,26 @@
...
 
     current_word = ""
     word_set = set()
 
     for char in input:
-        if char in word_chars:
-            current_word += char
+        if char not in boundary_chars:
+            if char in word_chars:
+                current_word += char
```
Comment (chloenott) [inline python/code_challenges/hashtable_repeated_word.py:16]: The word accumulator method!! Yes!! It's super cool to see this!

File: python/code_challenges/hashtable_repeated_word.py [hunk 2]
```diff
@@ -2,21 +2,26 @@
...
         else:
             if len(current_word) > 0:
                 if current_word in word_set:
                     return current_word
                 else:
                     word_set.add(current_word)
                     current_word = ""
+    if len(current_word) > 0:
+        if current_word in word_set:
+            return current_word
```
Comment (chloenott) [inline python/code_challenges/hashtable_repeated_word.py:26]: Good job handling this edge case after the for loop is complete!

**Review by chloenott — COMMENTED** [review]
This method is super crafty!!! Super cool sweeping through the input text character by character, building up each new word in an accumulator, and checking if that word is in the set already. Each action feels very deliberate. I look forward to checking out the whiteboard/readme when they're complete!
